src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -114,6 +115,14 @@ export class AddBulkConcepts extends Component {
     this.setState({ openResultModal: false });
   }
 
+  onKeyDown = (event, onKeyDown, onChange) => {
+    if (String(e.key).toUpperCase() === 'ENTER') {
```

Comment:
What happens when you need to change from ENTER to ENTRE?

Reply:
Please clarify, I don't understand what you mean by `ENTRE`

Reply:
What happens if you discover that you misspelt ENTER?

Reply:
I have updated it by setting a constant `const KEY_CODE_FOR_ENTER = 13;` that I use to Identify the key code and also gave it a descriptive name for a new user